debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerHelper.java

```diff
@@ -462,6 +462,12 @@ public static void setLogFilesForMining(OracleConnection connection, Scn lastPro
 
         List<LogFile> logFilesForMining = getLogFilesForOffsetScn(connection, lastProcessedScn, archiveLogRetention, archiveLogOnlyMode);
         if (!logFilesForMining.stream().anyMatch(l -> l.getFirstScn().compareTo(lastProcessedScn) <= 0)) {
+            Scn minScn = logFilesForMining.stream().map(LogFile::getFirstScn).min(Scn::compareTo).orElse(Scn.NULL);
```

Comment:
Tiny nit: I'd suggest to have one stream op per line for better debuggability:

```
logFilesForMining.stream()
    .map(LogFile::getFirstScn)
    .min(Scn::compareTo)
    .orElse(Scn.NULL);
```

I'll make that change and merge. Thx!